### PR TITLE
Implement conversion service

### LINKED ISSUE #150: As developer I want a conversion service for StUF-BG to API
The conversion service converts:

* API-requests to StUF-BG-requests
* StUF-BG-responses to API-responses

**Tasks**

* [x] Create endpoints `/convert/api2stufbg`
* [x] Create endpoints `/convert/stufbg2api`

### PR DESCRIPTION
**Pull request by shea-maykinmedia**
Fixes #150 

API-request to Stuf BG request endpoints:
/api/convert/api2stufbg/ingeschrevenpersonen/{{ bsn }}
/api/convert/api2stufbg/ingeschrevenpersonen?{{ filter }}={{ value }}
/api/convert/api2stufbg/ingeschrevenpersonen/{{ bsn }}/kinderen
/api/convert/api2stufbg/ingeschrevenpersonen/{{ bsn }}/ouders
/api/convert/api2stufbg/ingeschrevenpersonen/{{ bsn }}/partners

Stuf BG response to API response endpoints:
/api/convert/stufbg2api/ingeschrevenpersonen
/api/convert/stufbg2api/ingeschrevenpersonen/{{ bsn }}/kinderen
/api/convert/stufbg2api/ingeschrevenpersonen/{{ bsn }}/ouders
/api/convert/stufbg2api/ingeschrevenpersonen/{{ bsn }}/partners

I also have a postman collection that I can give if you are interested.

### CODE REVIEW

**Review by joeribekker — CHANGES_REQUESTED** [review]
This conversion cannot be part of the same API spec as the BRP API and i miss documentation on this feature :)

**Comment by shea-maykinmedia** [comment]
Convert endpoints have been moved out of the BRP endpoints and documentation can be found here https://github.com/maykinmedia/open-personen/blob/feature/converter-endpoints/docs/convert/index.rst.

**Review by joeribekker — APPROVED** [review]

File: docs/convert/index.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.. _convert_index:
+
+==================
+Convert Service
```
Comment (joeribekker) [inline docs/convert/index.rst:4]: Conversion service

File: docs/convert/index.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+==================
+
+The conversion service allows converting Open Personen API requests to StUF-BG requests and
```
Comment (joeribekker) [inline docs/convert/index.rst:7]: "converting Open Personen API requests" -> "converting BRP API requests"

File: docs/convert/index.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+StUF-BG responses to Open Personen API responses.  The conversion service does not make any
```
Comment (joeribekker) [inline docs/convert/index.rst:8]: "to Open Personen API responses" -> "to BRP API responses"

**Pull request merged**